lib/index.js

```diff
@@ -91,7 +85,7 @@ Hannibal.prototype.create = function buildValidator (schema) {
       transforms: self.transforms,
       fetchValidator: self.fetchValidator.bind(self),
       opts: opts || {},
-      originalData: lodash.clone(input)
+      originalData: Object.assign({}, input)
```

Comment:
As far as I can see we were always shallow cloning an plain object (`{}`) here, so no need for the large dependency